url_shortener/shorten_app/views.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.http import HttpResponse
+import json
+import random, string
+
+from .forms import EnterUrlForm
+from .models import Url
+
+
+def redirect(request, url):
+    current_obj = Url.objects.filter(short_url=url)
+    if len(current_obj) == 0:
+        return render(request, 'Error here')
+    context = {'obj':current_obj[0]}
+    return render(request, 'redirect.html', context)
```

Comment:
Already starting to forget Django a bit but is there supposed to be a redirect returned to the long URL saved at current_obj here if len(current_obj) != 0?